packages/gestalt/src/icons-vr-theme/compact/index.ts

```diff
@@ -10,6 +10,7 @@ import compactChevronUp from './compact-chevron-up.svg';
 import compactDash from './compact-dash.svg';
 import compactInfoCircleFill from './compact-info-circle-fill.svg';
 import compactLock from './compact-lock.svg';
+import compactSparkle from './compact-sparkle.svg';
```

Comment:
If it's not too much - could you also paste this into the Android package as well. Thank you